GOVERNANCE.md

```diff
@@ -48,6 +48,25 @@ There are three categories of project membership:
    merge/close PRs, but only per the group's approval. See the [OWNERS](OWNERS)
    file for the current list of Admins.
 
+### Admins
+
+Since the role of an 'Admin' is mainly administrative, the list of Members
+within this group should not need to change regularly. The following rules
+govern adding and removing Admins:
+
+- New Admins can be proposed via a PR to edit the [OWNERS](OWNERS) file. Normal
+  PR voting rules apply.
+- Admins can be removed via a PR to edit the [OWNERS](OWNERS) file. Normal
+  PR voting rules apply.
+- Admins can request to be removed via a PR to edit the [OWNERS](OWNERS) file.
+  Since the group can not force an Admin to continue in that role, the PR
+  does not following the normal voting rules and can be merged by any of the
+  Admins, including the person requesting to be removed.
+- If a PR to edit the list of Admins would result in there not being any
```

Comment:
Unlikely but the last admin could self-remove themselves per the previous rule. Might be preferable for an ongoing admin to always merge instead of self-merge. Last admin standing needs to handle the state of the project. :-)

Reply:
I thought about that but then what if no other admin merges?  :-)   I kind of figured the last bullet implies the "last admin standing" stuff you're talking about.  But see if my latest edits to the previous bullet helps